extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -360,19 +360,17 @@ public function getPage() {
 					case NS_CATEGORY:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this "switch/case" number of lines from 17 to at most 10, for example by extracting code into function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1151)